security/security.manager.ts

```diff
@@ -0,0 +1,111 @@
+import { HttpBadRequestError } from '@errors/http';
+import { getEnv } from '@helper/environment';
+import { IconikToken, IconikTokenModel, IconikTokenSchema } from '@models/DynamoDB/iconik-token.model';
+import { IconikService } from '@workflowwin/iconik-api';
+import { MetadataFieldSchema } from '@workflowwin/iconik-api/dist/src/metadata/metadata-methods';
+import { CustomActionSchema } from '@workflowwin/iconik-api/src/assets/assets-methods';
+import { WebhookResponseSchema } from '@workflowwin/iconik-api/src/notifications/notifications-methods';
+import { ScanResponse } from 'dynamoose/dist/DocumentRetriever';
+import { SecurityService } from './security.service';
+
+const refreshHoursField: MetadataFieldSchema = {
+  field_type: 'integer',
+  name: 'Set Token Refresh Time',
+  description: 'WIN Automation View',
+  required: true,
+};
+
+export class SecurityManager {
+  private readonly service: SecurityService;
+
+  constructor() {
+    this.service = new SecurityService();
+  }
+
+  async initialization(iconikService: IconikService) {
+    try {
+      const apiUrl = getEnv('CHANGE_REFRESH_TOKEN_PERIOD_CA_URL');
+
+      const field: MetadataFieldSchema = await iconikService.metadata.createMetadataField(refreshHoursField);
+      const view = await iconikService.metadata.createMetadataView({
+        name: 'win_RefreshHoursView',
+        label: 'Set Token Refresh Time',
+        view_fields: [field],
+      });
+      await iconikService.metadata.addCategoryToView(view.id, 'custom_actions');
+
+      const customAction: CustomActionSchema = await iconikService.assets.createCustomAction('NONE', {
+        context: 'NONE',
+        type: 'POST',
+        title: 'Change Refresh Token Period',
+        url: apiUrl,
+        metadata_view: view.id,
+      });
+
+      await this.service.removeAclFromCA(iconikService, customAction);

```

Comment:
Cloud you describe what problem do you try to solve?

Reply:
We allow seeing this CA only for the admin user who is the owner of the application.

Reply:
@khoroshilov-flo Can I resolve it?